packages/react-instantsearch/src/components/Breadcrumb.test.js

```diff
@@ -60,11 +60,11 @@ describe('Breadcrumb', () => {
     );
 
     const items = wrapper.find('.ais-Breadcrumb__itemLink');
-    expect(items.length).toBe(4); //Home + Breadcrumb (3 levels)
+    expect(items.length).toBe(4);
 
     items.first().simulate('click');
     expect(refine.mock.calls.length).toBe(1);
-    expect(refine.mock.calls[0][0]).toEqual(); //Home = empty refinement
+    expect(refine.mock.calls[0][0]).toEqual();
```

Comment:
I don't understand why you're calling `toEqual()` without any arguments. What about using [`.toBeNull()`](http://facebook.github.io/jest/docs/expect.html#tobenull)